storage/innobase/srv/srv0srv.cc

```diff
@@ -2756,6 +2807,9 @@ DECLARE_THREAD(srv_master_thread)(
 		}
 	}
 
+	/* Enable temporary tablespace encryption if set */
+	srv_enable_temp_encryption_if_set();
```

Comment:
Why here - at the start of shutdown? Especially when the called function exits first thing in this srv_shutdown_state

Reply:
hmm... I was pretty sure I was putting it inside of the loop :)

Reply:
Yet the current testcase succeeds - maybe need a new one where the server is started with off and switched to on dynamically?